### PR TITLE
added a new idea

### PR DESCRIPTION
**Pull request by jonathanlindev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 Add your ideas below!
 
 * Wobbly physics
-* Make a page that includes a timeline of how far the website has gone
\ No newline at end of file
+* Make a page that includes a timeline of how far the website has gone
+* A recap of our meetings / VoD / Transcript for future group members to see
```
Comment (tealleaf) [inline README.md]: Jonathan please add periods to all of the sentences

**Review by tealleaf — CHANGES_REQUESTED** [review]
Refer to comment on adding periods

File: README.md (hunk 1 above)
Comment (jonathanlindev) [reply README.md]: working on it

**Review by tealleaf — APPROVED** [review]
Looks good to me!

**Pull request merged**